dataactcore/scripts/load_duns_exec_comp.py

```diff
@@ -97,10 +98,10 @@ def load_from_sam(data_type, sess, historic, local=None, metrics=None, reload_da
         # a bit redundant but also date validation
         load_date = datetime.datetime.strptime(reload_date, '%Y-%m-%d').date()
     else:
-        sam_field = DUNS.last_sam_mod_date if data_type == 'DUNS' else DUNS.last_exec_comp_mod_date
```

Comment:
Making a note for a later ticket to rename `DUNS` file type string to something like `Recipient` for consistency. Not required for this ticket though.